--- libselinux-2.0.94/src/Makefile.orig	2010-08-06 13:34:59.502919908 +0200
+++ libselinux-2.0.94/src/Makefile	2010-08-06 13:35:16.906669822 +0200
@@ -9,7 +9,7 @@ 
 PYTHONLIBDIR ?= $(LIBDIR)/$(PYLIBVER)
 RUBYLIBVER ?= $(shell ruby -e 'print RUBY_VERSION.split(".")[0..1].join(".")')
 RUBYPLATFORM ?= $(shell ruby -e 'print RUBY_PLATFORM')
-RUBYINC ?= $(LIBDIR)/ruby/$(RUBYLIBVER)/$(RUBYPLATFORM)
+RUBYINC ?= $(LIBDIR)/ruby/$(RUBYLIBVER)
 RUBYINSTALL ?= $(LIBDIR)/ruby/site_ruby/$(RUBYLIBVER)/$(RUBYPLATFORM)
 LIBBASE=$(shell basename $(LIBDIR))
 
